Cargo.toml

```diff
@@ -7,6 +7,7 @@ authors = ["Wei Tang <hi@that.world>"]
 repository = "https://github.com/paritytech/libsecp256k1"
 keywords = ["crypto", "ECDSA", "secp256k1", "bitcoin", "no_std"]
 edition = "2018"
+resolver = "2"
```

Comment:
This should be enough. Please revert the rest of the changes.

Reply:
Done.